src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -86,6 +86,14 @@ public void Add(CookieCollection cookies)
             }
         }
 
+        public bool IsReadOnly
+        {
+            get
+            {
+                return false;
```

Comment:
@CIPop @stephentoub Thanks ,false it is then. I am closing https://github.com/dotnet/corefx/issues/12959